internal/services/instances.go

```diff
@@ -437,3 +468,27 @@ func setServiceStatusAndCustomValues(service *models.Service,
 	}
 	return nil
 }
+
+func getValue(values chartutil.Values, key string, maybetable bool) (string, error) {
+	value, err := values.PathValue(key)
+	if err == nil {
+		data, err := json.Marshal(value)
+		if err != nil {
+			return "", err
+		}
+		valueAsString := string(data)
+		return valueAsString, nil
+	}
+	if maybetable {
```

Comment:
Is it ok to look for this `maybetable` if there was previously an error?

Reply:
Yes. As I found, `PathValue` may generate an error where `Table` does not.